modules/social_features/social_core/social_core.module

```diff
@@ -990,6 +991,25 @@ function social_core_form_node_form_alter(&$form, FormStateInterface $form_state
   }
 }
 
+/**
+ * Implements hook_form_alter().
+ */
+function social_core_form_alter(array &$form, FormStateInterface $form_state, string $form_id): void {
+  if (method_exists($form_state->getFormObject(), 'getEntity')) {
```

Comment:
@nechai This code will move the comments field to the Settings group only if `$form['advanced']` does not exist. Please consider this. And add a comment with clarification, please.